minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -19,13 +19,7 @@
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.apache.accumulo.fate.util.UtilWaitThread.sleepUninterruptibly;
 
-import java.io.BufferedReader;
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import java.io.*;
```

Comment:
The travis CI build failed because of this start import.

https://travis-ci.org/apache/accumulo/jobs/472734911#L2367

Reply:
One thing I usually do locally is run `mvn verify -DskipITs`.  This will run most of the build checks on your code except for the integration test which take hours to run.

Reply:
Are you running maven version 3.1.1?

Reply:
FWIW, I'm using Maven 3.5.4.